uint8 device_id
uint8 instruction
float32[] payload